src/goSuggest.ts

```diff
@@ -153,5 +210,58 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				});
 			});
 		});
+		return Promise.all([pkgPromise, configPromise]).then(() => {
+			return Promise.resolve();
+		}); ;
+	}
+
+	// Return importable packages that match given word as Completion Items
+	private getMatchingPackages(word: string): vscode.CompletionItem[] {
+		if (!word) return [];
+		let completionItems = this.pkgsList.filter((pkgInfo: PackageInfo) => {
+			return pkgInfo.name.startsWith(word);
+		}).map((pkgInfo: PackageInfo) => {
+			let item = new vscode.CompletionItem(pkgInfo.name, vscode.CompletionItemKind.Keyword);
+			item.detail = 'Add import';
```

Comment:
I think this is going to need to include the pull import path name as additional context.  I think being clear that this will "Add an import" is also helpful.